README.md

```diff
@@ -46,130 +46,68 @@ Bazel manages all of the dependencies for you (including go and necessary compil
 
 ```
 bazel run //beacon-chain --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
+  --datadir /path/to/beacondatadir \
   --rpc-port 4000 \
-  --validator
-```
-
-This will spin up a full beacon node that connects to your running geth node, opens up an RPC connection for sharding validators to connect to it, and begins listening for p2p events.
-
-To try out the beacon node in development by simulating incoming blocks, run the same command above but enable the `--simulator` and a debug level, log verbosity with `--verbosity debug` to see everything happening underneath the hood.
-
-```
-bazel run //beacon-chain --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-  --validator \
+  --genesis-json /path/to/genesis.json \
   --simulator \
-  --verbosity debug
-```
+  --demo-config
 
-Now, deposit ETH to become a validator in the contract using instructions [here](https://github.com/prysmaticlabs/prysm/blob/master/docs/VALIDATOR_REGISTRATION.md)
+```
 
-If you don't want to deposit ETH and become a validator, one option is to run a beacon node as an Observer. A beacon observer node has full privilege to listen in beacon chain and shard chains activities, but it will not participate
-in validator duties such as proposing or attesting blocks. In addition, an observer node doesn't need to deposit 32ETH. To run an observer node, you discard the `--validator` flag.
+We added a `--simulator` flag that simulates other nodes connected to you sending your node blocks for processing. Given this is a local development version and you'll only be running 1 validator client, this gives us a good idea of what the system will need to handle in the wild and will help advance the chain.
 
-```
-bazel run //beacon-chain --\
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-```
+We also have a `--demo-config` flag that configures some internal parameters for you to run a local demo version of the system.
 
-### Running via Docker
+If you want to see what's happening in the system underneath the hood, add a `--verbosity debug` flag to show every single thing the beacon chain node does during its run time.
 
-To run the beacon node within a docker container, use the `//beacon-chain:image` target.
+## Running a Single, ETH2.0 Validator Client
 
-```text
-bazel run //beacon-chain:image --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-  --simulator \
-  --verbosity debug
-```
+Once your beacon node is up, you'll need to attach a validator client as a separate process. This validator is in charge of running Casper+Sharding responsibilities (shards to be designed in phase 2). This validator will listen for incoming beacon blocks and shard assignments and determine when its time to perform attester/proposer responsibilities accordingly.
```

Comment:
shards to be designed in phase 1 https://github.com/ethereum/wiki/wiki/Sharding-roadmap

the difference is phase 1 is shards data layer, phase 2 is shards execution layer